standards/management-access.md

```diff
@@ -0,0 +1,38 @@
+---
+expires: 2019-03-15
+---
+# Management access
+
+## The base principle
+
+Management or administrative access **must** be limited to authorised authenticated users and utilise multi-factor authentication wherever possible.
+
+## Secure Shell (SSH)
+
+Use of bastion or 'jump' boxes for access into systems is a useful technical security design that also helps 'choke' and control such sessions.
+
+Through immutable infrastructure and server design, state-less cluster expansion/contraction and automated diagnostic data capture the need to SSH into a server/node should be increasingly less common.
+
+It should be exceptional for an individual to login to a server/node via SSH and execute commands with elevated privledges (typically, `root`).
+
+### Using SSH
+
+SSH must be strictly controlled, and environments should be segregated so that no single bastion or 'jump' SSH server can access both production and non-production accounts.
+
+SSH shells must be limited to users who need shell (by comparison to users who will use SSH as a port forwarding tunnel).
+
+Joiners/Movers/Leavers processes must be strictly enforced (optimally, automated) on SSH servers as they are a critical and priviledged access method.
+
+SSH should not be password-based, and should use individually created and purposed SSH keypairs. <u>Private keys must not be shared or re-used</u>.
+
+## Application program interface (**API**)
```

Comment:
As this is preferred, maybe it should be first?

Reply:
PR inbound